### PR TITLE
added Home page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
     <title>React App</title>
```
Comment (AbhishekkGautam) [inline public/index.html]: we can replace the title `React App` with the app name.

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,13 @@
+import { Route,Routes } from "react-router-dom";
 import "./App.css";
-import logo from "./logo.png";
+import { Home } from "./pages/home/home"
```
Comment (AbhishekkGautam) [inline src/App.js]: we should use barrel exports. create an `index.js` inside `pages` folder & export `home` page from it.

File: src/pages/home/home.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+}
+
+
+ .header-nav {
+    background-color: var(--pink-dark);
+
+}
+.header-nav{
+    position: fixed;
+    z-index:555;
```
Comment (AbhishekkGautam) [inline src/pages/home/home.css]: we should avoid giving random z-index

File: src/pages/home/home.css [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    padding-right:10px;
+}
+
+.col1 i{
+    padding-right:10px;
+ 
+}
+
+
+@media only screen and (max-width:1300px){
```
Comment (AbhishekkGautam) [inline src/pages/home/home.css]: remove unused media query

File: src/pages/home/home.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+        height:auto;
+    }
+
+    .btn-login{
+        font-size: 1rem;
+        margin-right:3rem;
+    }
+    .col1 ul li{
+        font-size: 1rem;
+        padding:1rem 0rem 1rem 0rem;
```
Comment (AbhishekkGautam) [inline src/pages/home/home.css]: we can also write it like this -  `padding: 1rem  0`

File: src/pages/home/home.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+// import { videos } from '../../backend/db/products';
+import { Navbar } from '../../components/allComponents';
+import "./home.css";
+import { products } from '../../backend/db/products';
```
Comment (AbhishekkGautam) [inline src/pages/home/home.js]: we don't need to import `products` db from backend

File: src/pages/home/home.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+        <div className='btn-category'>
+          <button>All</button>
+          <button>Ice Cream</button>
+          <button>Cake</button>
+          <button>Choclate</button>
+        </div>
+        <div className='videosListing'>
+
+        {
+          products.map(item=>{
```
Comment (AbhishekkGautam) [inline src/pages/home/home.js]: I think we're not supposed to get products data like this. we should fetch data using API. On Mockbee docs, we have routes defined for each data. so we are supposed to fetch the data & store it in context ( state ) & then use it in our components

**Review by AbhishekkGautam — CHANGES_REQUESTED** [review]

File: src/index.css [hunk 8]
```diff
@@ -1,13 +1,37 @@
...
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+body{
+  font-family: 'Roboto Condensed', sans-serif;
+}
+a{
```
Comment (muskaanshah) [inline src/index.css]: Why a tag?

File: src/index.css [hunk 9]
```diff
@@ -1,13 +1,37 @@
...
+  text-decoration: none;
+}
+li{
+  list-style: none;
 }
+
+:root{
```
Comment (muskaanshah) [inline src/index.css:2]: Add root to the top of the file

File: src/pages/home/home.css (hunk 4 above)
Comment (muskaanshah) [inline src/pages/home/home.css]: Remove this

**Review by muskaanshah — APPROVED** [review]
Design suggestions -
chocolate spelling incorrect
padding is inconsistent, instead of giving specific width, give specific padding, so it's scalable
add NavLink fr drawer links

**Review by samyakshah3008 — APPROVED** [review]
Overall review: Good naming conventions, easy and simple to understand, clean UI 
Suggestions: You can add comments in code where necessary for better code review experience
Also you can write more in PR preview description 
Try to use max or min width for responsibility 
Use format document to make code looks prettier

Rest is all good for approval ✅

File: src/pages/home/home.js (hunk 6 above)
Comment (JuvitaSaini09) [reply src/pages/home/home.js]: updated ,thnks for the review

File: src/pages/home/home.js (hunk 7 above)
Comment (JuvitaSaini09) [reply src/pages/home/home.js]: I have updated it ,thankyou for letting me

File: src/index.css (hunk 8 above)
Comment (JuvitaSaini09) [reply src/index.css]: i will update it thnks for letting me know

File: src/index.css (hunk 9 above)
Comment (JuvitaSaini09) [reply src/index.css:2]: i have updated it,thankyou for the suggesstion

File: src/pages/home/home.css (hunk 4 above)
Comment (JuvitaSaini09) [reply src/pages/home/home.css]: Thankyou for the reviews

**Pull request merged**